autora/cycle/_simple_cycle.py

```diff
@@ -136,15 +196,15 @@ class _SimpleCycle:
         We can also run the cycle with more control over the execution flow:
         >>> next(cycle) # doctest: +ELLIPSIS
         Generated 4 theories
-        <_simple_cycle._SimpleCycle object at 0x...>
+        <simple.SimpleCycle object at 0x...>
 
         >>> next(cycle) # doctest: +ELLIPSIS
         Generated 5 theories
-        <_simple_cycle._SimpleCycle object at 0x...>
+        <simple.SimpleCycle object at 0x...>
 
         >>> next(cycle) # doctest: +ELLIPSIS
         Generated 6 theories
-        <_simple_cycle._SimpleCycle object at 0x...>
+        <simple.SimpleCycle object at 0x...>
 
         We can continue to run the cycle as long as we like,
         with a simple arbitrary stopping condition like the number of theories generated:
```

Comment:
not sure how to suggest a change to an unmodified line, but the second 'between' in the below should be 'of'. ie, '...difference between the gradients of...'

`... or the precision (here we keep iterating while the difference between the gradients
        between the second-last and last cycle is larger than 1x10^-3).`